packages/appstore/src/components/modals/account-type-modal/types.ts

```diff
@@ -20,7 +21,7 @@ export type TAccountCard = {
 };
 
 export type TTradingPlatformAvailableAccount = {
```

Comment:
@hamza-deriv What's the difference between this type and the one in `packages/appstore/src/types/common.types.ts`? 🤔

Reply:
they both are same

Reply:
There are many places where types are replicated 
we have to get rid of that when we are going to refactor the useStore thing